pkg/scheduler/framework/runtime/registry_test.go

```diff
@@ -93,22 +97,34 @@ func isRegistryEqual(registryX, registryY Registry) bool {
 		}
 	}
 
+	for name := range registryY {
```

Comment:
isn't this covered in the first loop above?

Reply:
whoops yes!